### ISSUE TITLE
[Feature] Breast/Cup Size Sort on Performers Page

### ISSUE THREAD

**Issue by UncertainMongoose** [stashapp/stash#4831]
### Is your feature request related to a problem? Please describe:

There is a `Penis Size` sort but not a `Breast Size` sort option on the `/Performers` page to be able to sort the women as well as the men by their physical attributes.

### Describe the solution you'd like:

I would like to see a `Breast Size` or `Cup Size` sort added to allow sorting by increasing or decreasing cup size.

### Describe alternatives you've considered:

I am not particularly well-versed in JS or CSS to but I'd imagine that this would be possible by grabbing the characters between the number and first hyphen in a female performer's `Measurements` field as a sort of "top level" sort to sort by cup size "category" first. Then the numbers preceding the letter(s) could be included for further sorting.

**Comment by feederbox826** [comment]
I made a plugin to add cup size tags to performers and realized there's a lot more to this....

- there's discource around DD=E and DDD=F
- bra sister sizes https://github.com/stg-annon/performerBodyCalculator/pull/2

imo this would best be left for a plugin to calculate by volume rather than just cup size since it's a ordinal (comparison) category rather than penis size which is a ratio category [^1](https://en.wikipedia.org/wiki/Level_of_measurement) with an absolute zero

**Comment by stg-annon** [comment]
yeah this was kind of the motivation behind [performerBodyCalculator](https://github.com/stg-annon/StashScripts/tree/main/plugins/performerBodyCalculator) beyond adapting an existing script that kind of did the same thing but attempts to remain configurable

> imo this would best be left for a plugin to calculate by volume rather than just cup size

@melon-scientist was the one that helped [with that part](https://github.com/stg-annon/performerBodyCalculator/pull/2)

of course none of this really helps with "sorting" but filtering

**Comment by feederbox826** [comment]
Related: #1661

**Comment by GalacticCheat** [comment]
Maista sucks cock
![Image](https://github.com/user-attachments/assets/ed12369e-cb07-434a-b49f-fbc470272035)
https://i.ibb.co/ds49xcXt/image-2-copie.jpg
